pyteal/ast/assert_.py

```diff
@@ -11,23 +12,29 @@
 class Assert(Expr):
     """A control flow expression to verify that a condition is true."""
 
-    def __init__(self, cond: Expr) -> None:
+    def __init__(self, cond: Expr, *condMulti: List[Expr]) -> None:
```

Comment:
nit: I wonder what if we just keep only one argument: `__init__(self, *conds: Expr): -> None`? I think this is not breaking the old code, wile still introducing the new feature of multi-expression clause.

Reply:
also, the type annotation seems off, maybe you just want `*conds: Expr`? by https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values

Reply:
Thanks for pointing out the type annotation error! I expected the behavior to be similar to JavaScript's. I thought adding an argument list instead of replacing the existing one gave better suggestions for recommended usage.

With `__init__(self, *conds: Expr): -> None`, the user sees no type error if they were to write `Assert()`. However, if the first positional argument is included, it suggests that the first argument is required and the remaining ones are optional – `Assert()` would show a warning because it's missing a required argument.

Reply:
👍